apps/src/p5lab/AnimationTab/ItemLoopToggle.jsx

```diff
@@ -30,21 +30,15 @@ class ItemLoopToggle extends React.Component {
     return (
       <OverlayTrigger overlay={tooltip} placement="bottom" delayShow={500}>
         <div
-          style={[styles.loopToggleStyle, this.props.style]}
+          style={{...this.props.style}}
+          className={style.loopToggle}
           onClick={this.toggleClicked}
         >
-          <img src={iconImageSrc} style={styles.loopIconStyle} />
+          <img src={iconImageSrc} className={style.loopIconStyle} />
```

Comment:
I don't see `loopIconStyle` anywhere (in previous version or new version), am I missing something?

Reply:
Huh, good point. I think I was on autopilot for this change and missed that. Seems safe enough to just remove this.